src/styles/index.scss

```diff
@@ -1,3 +1,226 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin: 0;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be no more than 1 BEM block per file. The `.header` block should be in its own SCSS file.